Update new-relic-api-keys.mdx (#18880)
newrelic/docs-website: src/content/docs/apis/intro-apis/new-relic-api-keys.mdx

## View and manage API keys [#keys-ui]

> You can manage most API keys from the [API keys UI page](https://one.newrelic.com/api-keys). To find that: from the [user menu](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings/#user-menu), click <DNT>**API keys**</DNT>.

Author: This one looks good!

> To view a key you will need to copy the key ID which you will need to view the key via Nerdgraph, which you can also [use to manage keys](/docs/apis/nerdgraph/examples/use-nerdgraph-manage-license-keys-user-keys)

Reviewer: ```suggestion
To view a key in NerdGraph, copy the key ID.  You can also [use NerdGraph to manage keys](/docs/apis/nerdgraph/examples/use-nerdgraph-manage-license-keys-user-keys)
```

Author: This looks good!

You should treat your API keys securely, as you would passwords and other sensitive information. Some recommendations:

* For the license key and the browser key, consider implementing a key rotation strategy: creating new keys and deleting old ones on a set schedule. Considerations:

>   * You can't delete the original ingest keys associated with an account, so to implement a strong security strategy, you must create additional ingest keys that you can later delete.

Reviewer: ```suggestion
  * You can't delete the original ingest keys associated with an account, so we recommend creating additional ingest keys that you can later delete. This ensures you're assuming a strong security posture. 
```

Author: This looks good!

  * Note that this doesn't apply to the mobile app token; you can't delete a token or create additional tokens.
* For the <InlinePopover type="userKey"/>:
  * Instruct your team members to keep their user keys secure.

>   * When members leave your organization, remove them from New Relic. This disables all the user keys associated with their user ID. 

Reviewer: ```suggestion
  * When members leave your organization, remove their user IDs from New Relic. This disables all the user keys associated with their user ID. 
```

Author: This looks good!

>   * When members leave your organization, remove them from New Relic which will disable all the user keys assocaited with their user ID

Reviewer: ```suggestion
  * When members leave your organization, remove them from New Relic which will disable all the user keys assocaited with their user ID.
```

Author: This one looks good!

>   * When members leave your organization, remove them from New Relic which will disable all the user keys assocaited with their user ID

Reviewer: ```suggestion
  * When members leave your organization, remove them from New Relic. This disables all the user keys associated with their user ID. 
```
